cmd/kaspawallet/daemon/server/create_unsigned_transaction.go

```diff
@@ -89,7 +89,7 @@ func (s *server) createUnsignedTransactions(address string, amount uint64, fromA
 func (s *server) selectUTXOs(spendAmount uint64, feePerInput uint64, fromAddresses []*walletAddress) (
 	selectedUTXOs []*libkaspawallet.UTXO, changeSompi uint64, err error) {
 
-	selectedUTXOs = []*libkaspawallet.UTXO{}
+	selectedUTXOs = make([]*libkaspawallet.UTXO, 0, len(s.utxosSortedByAmount))
```

Comment:
Please undo this. 
Usually selectedUTXOs is not going to include all of s.utxosSortedByAmount, often just a tiny fraction, so you are over-allocating here.

Since it's not known ahead-of-time how much to allocate, better let append do it's thing.